client/client.go

```diff
@@ -22,12 +22,12 @@ import (
 
 // Client provides the interface for the consuming application
 type Client struct {
```

Comment:
if those changes are not mandatory for our PR, I suggest we discard using git checkout